docs/docs/guides/authentication/cloudentity.md

```diff
@@ -0,0 +1,110 @@
+
+On Cloudentity's platform, go to **Enforcement > Authorizers**, click **CREATE GATEWAY**, and from the list that's displayed choose Kusk.
+
+![cloudentity authorizers list](./img/cloudentity-authorizers.png)
+
+### 2. Install Cloudentity's authorizer in your cluster
+
+The bottom of the quickstart section contains a `helm` command that installs Cloudentity's authorizer and configures with credentials that are used to pull authorization policies from Cloudentity. 
+
+Be sure to click on "Show" before copying the command. 
+
+![cloudentity helm command](./img/cloudentity-helm-command.png)
+
+Once copied, run the `helm` command in your terminal. 
+
+### 3. Deploy an API with Kusk that uses Cloudentiy's authorizer
+
+This is an example that uses a Kusk Policy with your OpenAPI definition to configure your gateway to use Cloudentity's authorizer with Kusk's mocking policy: 
+
+`cloudentity-api.yaml` 
+
+```yaml
+openapi: 3.0.0
+info:
+  title: super-simple
+  description: super simple api
+  version: '0.1.0'
+x-kusk:
+  mocking:
+    enabled: true
+  auth:
+    scheme: cloudentity
+    auth-upstream:
+      host:
+        hostname: cloudentity-authorizer-standalone-authorizer.kusk-system # default authorizer service, change in case your authorizer is installed differently
+        port: 9004
+paths:
+  /hello:
+    get:
+      responses:
+        200:
+          description: Return a hello world message
+          content: 
+            application/json:
+              schema:
+                type: object
+                properties:
+                  message:
+                    type: string
+              example:
+                message: Mocked Response! 
+```
+
+Create the file and deploy it to your cluster: 
+
+```bash
+kusk deploy -i cloudentity-api.yaml
+```
+
+Without any additional configuration, you should be able to call the API without it being blocked and Kusk will return a mock response as configured in the example field: 
+
+:::info
+Cloudentity's authorizer pulls new rules on an interval of 1 minute. During that period after the initial confiugration your will receive responses with HTTP Status Code `403`. 
+:::
+
+```bash
+curl $(kusk ip)
+
+{"message":"Mocked Response!"}
+```
+
+### 4. Add an API Authorization Policy with Cloudentity
+
+To see a simple example of how Cloudentity blocks an API, configure the simple authorization policy **"Block API"** with cloudentity to block all API calls. 
+
+For that, on Cloudentity's portal, navigate to **Enforcement > APIs** where you'll see the list of paths that Kusk has synchronized with your Cloudentity's account.
+
+Click on the policy column to add a **Block API** rule. 
+
+![cloudentity policy selection](./img/cloudentity-policy-selection.png)
+
+### 5. Test that API calls are blocked
+
+```
+curl $(kusk ip) -v
+...
+< HTTP/1.1 403 Forbidden
+< date: Tue, 04 Oct 2022 06:54:52 GMT
+< x-envoy-upstream-service-time: 9
+< server: envoy
+< content-length: 0
+```
+
+:::info
```

Comment:
Does this render correctly?

Don't you need an empty lines between `<contents>` in `:::info<contents>:::`?

Reply:
Never mind. It does render correctly. I forgot about preview.